src/components/ContextMenuItem.js

```diff
@@ -93,6 +96,12 @@ class ContextMenuItem extends Component {
                         wrapperStyle={styles.pr9}
                         success={this.props.isDelayButtonStateComplete}
                         description={this.props.description}
+                        descriptionTextStyle={styles.breakAll}
+                        style={
+                            this.props.windowWidth > variables.tabletResponsiveWidthBreakpoint
```

Comment:
If we use `tabletResponsiveWidthBreakpoint`,  `maxwidth` won't be set for contextmenuitem between `mobileResponsiveWidthBreakpoint` and `tabletResponsiveWidthBreakpoint`.

https://user-images.githubusercontent.com/25876548/233104426-ef28981f-7cdf-4d05-b4a8-0a77f728358b.mov

Reply:
We might have to create three different files 'BaseContextMenuItem.js', 'index.js' and 'index.native.js' similar to `AnchorForCommentsOnly` component. But lets wait for @cead22's suggestion before procedding.

Reply:
@hayata-suenaga @mountiny Any suggestion, on how to proceed. To handle iPad, we need to set different styles based on width of the device for native and web. So, I suggested to create two different files 'index.js' and 'index.native.js'. Should we proceed with my suggestion.

cc: @cead22

Reply:
@allroundexperts could you explain for me why we need to set `maxWidth` for `MenuItem`? why, for example, we cannot use `flex: 1`?

I realize this throughout this codebase in general and have wanted to ask this question 😅 (i'm still new to this repo)

Reply:
Bump on this @cead22 @hayata-suenaga @cead22

Reply:
@allroundexperts thank you for the explanation. that makes sense!